packages/connectivity/src/scp-cf/environment-accessor/xsuaa.ts

```diff
@@ -73,6 +39,8 @@ export function getXsuaaService(options?: {
   const cacheKey = `${credentials.clientid}:${disableCache}`;
 
   if (!xsuaaServices[cacheKey]) {
+    // XsuaaService is a class that extends Service from @sap/xssec and is a representation of XSUAA credentials
```

Comment:
```suggestion
    // XsuaaService is a class from @sap/xssec and is a representation of XSUAA credentials extracted from a reuse service, e.g., destination
```

[q] Do we care what it extends?